examples/nestjs/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "pact-nestjs-full-example",
+  "version": "1.0.0",
+  "scripts": {
+    "test:consumer": "cd ./consumer && npm run test",
+    "test:provider": "cd ./provider && npm run test",
+    "test": "npm run test:consumer && npm run test:provider"
```

Comment:
This actually breaks in CI, because we iterate over all examples and run `npm install`. I reckon we should just have `/examples/nest-js-consumer` and `/examples/nest-js-provider` or similar. What do you think?

Reply:
Resolved in the other branch